pipelines/tekton/azureml-container-pipeline/model-upload/Makefile

```diff
@@ -0,0 +1,9 @@
+create_pod:
+	cat basic-pod.yaml | sed 's/PVC_NAME/${PVC_NAME}/g' | oc create -f -
+	oc get pod -w basic-pod
+
+upload_model:
+	oc cp ${MODEL_PATH} basic-pod:/workspace/${PVC_NAME}/
+
+clean:
+	cat basic-pod.yaml | sed 's/PVC_NAME/${PVC_NAME}/g' | oc delete -f -
```

Comment:
_"All of the targets here should be declared as `.PHONY` targets, since the target names are not file names that the targets will create. See [here](https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html) for more info."_ - @grdryn circa 2023 (colorized) ([taken from here](https://github.com/opendatahub-io/ai-edge/pull/55#discussion_r1314857565))

Also,

_"This looks like a fine approach :+1: 
One possible alternative to using a ~~ConfigMap~~ `sed` would be an [OpenShift Template](https://docs.openshift.com/container-platform/4.13/openshift_images/using-templates.html) resource. [...] I think on OpenShift, a Template would be a more natural wrapper for resource definitions in the cluster that gets parameterized before being applied, but this approach is also fine for now."_ - @grdryn circa 2023 (colorized) ([taken from here](https://github.com/opendatahub-io/ai-edge/pull/55#discussion_r1314977981))